R/inven.R

```diff
@@ -48,13 +48,17 @@ inven <- function(x,obj,need=NULL,crump=TRUE) {
 #' }
 #' @return original mrgmod
 #' @export
-inventory <- function(x,obj,..., .strict = TRUE) {
+inventory <- function(x,obj,..., .strict = FALSE) {
```

Comment:
based on your comment below, this should be changed back to true (which is my preference anyway)